src/posts/vector-data-cubes/index.md

```diff
@@ -138,14 +144,25 @@ We can select points based on geometry and proximity to other points using the `
 max_temp_city = era5_europe_cities.isel(
     **era5_europe_cities["2m_temperature"].argmax(dim=["time", "geometry"])
 )
+
 max_temp_buffer = era5_europe_cities.xvec.query(
-    "geometry", max_temp_city["geometry"].data.item().buffer(0.5)
+    "geometry", max_temp_city["geometry"].data.item(), predicate="dwithin", distance=0.5
```

Comment:
In the context of both xvec and geopandas, using dwithin predicate shall be preferred over intersection via buffer as it can be significantly faster.